### PR TITLE
Add comment to `RRect`.

### PR DESCRIPTION
**Pull request by bernaferrari**
Is this something like you expected?

### CODE REVIEW

File: lib/ui/painting.dart [hunk 1]
```diff
@@ -5282,6 +5282,10 @@ class Canvas extends NativeFieldWrapperClass1 {
     _drawRRect(rrect._getValue32(), paint._objects, paint._data);
   }
 
+  // RadiusX and radiusY values, on every corner, must be finite.
+  // NaN values and infinities are not valid.
```
Comment (dnfield) [inline lib/ui/painting.dart:5286]: What happens when we give invalid data?

File: lib/ui/painting.dart [hunk 2]
```diff
@@ -5282,6 +5282,10 @@ class Canvas extends NativeFieldWrapperClass1 {
...
+  // Negative radii are interpreted as 0.
+  // Positive radii are interpreted as 0 < radii < float32.
```
Comment (dnfield) [inline lib/ui/painting.dart:5288]: I think you can just say `radii are clamped between 0..FLT_MAX` or something like that.

The problem ends up happening upsream of this method if someone provides a value > FLT_MAX.

**Comment by chinmaygarde** [comment]
@bernaferrari I believe the AI here is to just clearly say that exceeding FLT_MAX is undefined behavior.

**Comment by bernaferrari** [comment]
Thanks. I'll update shortly. My life is super hurry until next week!

**Comment by chinmaygarde** [comment]
Closing this as stale. Please re-open once progress can be made. Thanks!

**Pull request closed without merging**